src/app/basic/scrolling.component.ts

```diff
@@ -1,6 +1,8 @@
 import { Component, inject } from '@angular/core';
 import { FullEmployee } from '../data.model';
 import { DataService } from '../data.service';
+import { DatatableComponent } from '../../../projects/ngx-datatable/src/lib/components/datatable.component';
+import { DataTableColumnDirective } from '../../../projects/ngx-datatable/src/lib/components/columns/column.directive';
```

Comment:
Please import them from the public API with `from 'projects/ngx-datatable/src/public-api';` and not using the internal imports.

Reply:
done